WalletWasabi.Fluent/Models/Wallets/WalletModel.cs

```diff
@@ -108,22 +143,39 @@ public IAddress GetNextReceiveAddress(IEnumerable<string> destinationLabels)
 		return nextReceiveAddress;
 	}
 
-	public IWalletInfoModel GetWalletInfo()
+	private static string WalletFile(string walletDir, string walletWalletName) => Path.Combine(walletDir, walletWalletName + "." + WalletDirectories.WalletFileExtension);
```

Comment:
This is not something the UI should compute. Move it to where it belongs to.

Reply:
I agree.